installation.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+virtualenv -q -p /usr/bin/python3.5 scancode-server
+source scancode-server/bin/activate
+pip install -r requirements.txt
+echo "source scancode-server/bin/activate"
```

Comment:
Why this echo? what does it bring?

Reply:
that was a mistake. I'll remove it!! :wink: